lbrynet/core/LBRYMetadata.py

```diff
@@ -0,0 +1,135 @@
+import requests
+import json
+import time
+
+from copy import deepcopy
+from googlefinance import getQuotes
+from lbrynet.conf import CURRENCIES
+import logging
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+BITTREX_FEE = 0.0025
+
+SOURCE_TYPES = ['lbry_sd_hash', 'url', 'btih']
+BASE_METADATA_FIELDS = ['title', 'description', 'author', 'language', 'license', 'content-type', 'sources']
+OPTIONAL_METADATA_FIELDS = ['thumbnail', 'preview', 'fee', 'contact', 'pubkey']
+
+#v0.0.1 metadata
+METADATA_REVISIONS = {'0.0.1': {'required': BASE_METADATA_FIELDS, 'optional': OPTIONAL_METADATA_FIELDS}}
+#v0.0.2 metadata additions
+METADATA_REVISIONS['0.0.2'] = {'required': ['nsfw', 'ver'], 'optional': ['license_url']}
+CURRENT_METADATA_VERSION = '0.0.2'
```

Comment:
Add a version number inside the meta-data itself.  Without it, we'll end up trying to back-out what version we're working with and that will become a pain.

In the very near future we'll need to document what it means to change a version (one option: http://snowplowanalytics.com/blog/2014/05/13/introducing-schemaver-for-semantic-versioning-of-schemas/)

We should also consider using standardized json-schema validators: https://pypi.python.org/pypi/jsonschema